testing_and_setup/compass/utility_scripts/compare_fields.py

```diff
@@ -18,153 +21,153 @@
 args = parser.parse_args()
 
 if not args.filename1:
-	parser.error("Two filenames are required inputs.")
+    parser.error("Two filenames are required inputs.")
```

Comment:
Lots of tabs have been replaced with spaces.  It's a bad practice to have a mix of tabs and spaces in a python file, and generally tabs should be avoided.

Reply:
Thanks. I appreciate that.